src/pybind/mgr/dashboard/frontend/src/app/core/auth/login/login.component.ts

```diff
@@ -62,9 +65,16 @@ export class LoginComponent implements OnInit {
   }
 
   login() {
-    this.authService.login(this.model).subscribe(() => {
-      const url = _.get(this.route.snapshot.queryParams, 'returnUrl', '/');
-      this.router.navigate([url]);
-    });
+    this.authService
+      .login(this.model)
+      .pipe(mergeMap(() => this.settingsService.getValues('POST_INSTALLED')))
```

Comment:
Now I get it. So, for every user logging in, the process will become slightly delayed as it now requires an extra round-trip... right? 🙈 And that's why you were embedding the cluster-creation flag in the auth endpoint... Let's discuss this tomorrow at our daily.

Reply:
Done as discussed